facebook/ThreatExchange
# Move PHP code to api-examples folder

Moving the php cli code to be an api example for php.
This folder will need later work to update the api example
to the latest supported api for ThreatExchange.

Test Plan:

1. Set TX_APP_ID and TX_APP_SECRET for a test application in threat exchange.
2. cd to the `api-reference-examples/php`
3. Run `php search.php -t indicators -q blackbox` and confirm that executes okay.
